src/apps/chifra/internal/scrape/handle_scrape.go

```diff
@@ -115,12 +122,10 @@ func (opts *ScrapeOptions) HandleScrape() error {
 			if !ok {
 				break
 			}
-			goto PAUSE
```

Comment:
First. Thanks so much for your PR. The intention of this is that the scraper never quits. It pauses under the assumption that the node software's RPC has temporarily had a problem. In other words, even if the node software dies, I think the scraper should not exit it's loop.